packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.base.tsx

```diff
@@ -357,6 +367,10 @@ export class ColorPickerBase extends React.Component<IColorPickerProps, IColorPi
       this.setState({ color: newColor, editingColor: undefined });
     }
   }
+
+  private _getSliderValue(alpha: number): number {
```

Comment:
nit: the method name is getSliderValue, but this is used both to get A value from Slider and slider value from A. See if you can name better